backend/aws_helpers/dynamo_db_utils/dynamo_db_utils.py

```diff
@@ -97,6 +99,24 @@ def get_dynamo_items_by_gsi(table_name: str, gsi_value: Union[int, str]) -> list
     return response_items
 
 
+def to_decimal(o):
```

Comment:
Should add some pydocs to describe this utility function @dwu359

Reply:
Yes, but the function will probably be rewritten anyways since there are more efficient ways to accomplish that also

Reply:
so will that be in this PR or separate/followup PR?

Reply:
@dwu359